javalib/src/main/scala/java/lang/dec2flt/Tables.scala

```diff
@@ -0,0 +1,72 @@
+package java.lang.dec2flt
+
+import java.math.BigInteger
+
+private[dec2flt] object Tables {
+  /** Pre-computed power-of-tens in 64-bit extended precision floating point number.
+   *
+   *  The entries of this table has an exact floating point values:
+   *  10^e = 5^e * 2^e, and the 2^e part can be absorbed into the binary exponent.
+   *  5^e < 2^64 <=> e < log5(2^64) <=> e < 27.563299716697156...
+   */
+  val smallPowerOfTens = Array[FloatingPoint](
+    FloatingPoint.normalized(new BigInteger("8000000000000000", 16), -63), // 0
```

Comment:
It seems all the constants here are unsigned `Long` values. Store them as such, and convert them to `BigInteger` without having to parse a string?

This can be done with a small helper like
```scala
def ulongToBigInt(x: Long): BigInteger =
  if (x >= 0L) BigInteger.valueOf(x)
  else BigInteger.valueOf(x & ~Long.MinValue).setBit(63)
```